### DISCUSSION TITLE
AlpineJS ignores escape quotes

### DISCUSSION THREAD

**Discussion by GeorgeFlorian** [alpinejs/alpine#4070] [1. Help]
I'm using AlpineJS with Django Templates and I can't escape quotes from an address:
```
Alpine Expression Error: missing ) after argument list

Expression: "radioAddressOnClick('L'Address', '2 rue de l'outil', '12345', 'Brasov', 'Romania' )"
```
For the element:
```django
<input type="radio"
       name="delivery_address_radio"
       id="delivery-{{ address.id }}"
       value="{{ address.delivery_address }}"
       class="hidden peer/delivery_address"
       {% if default_delivery_address.delivery_address == address.delivery_address %}
         checked
       {% endif %}
       @click="radioAddressOnClick(`{{ address.name|escapejs }}`, `{{ address.delivery_address|escapejs }}`, `{{ address.postal_code }}`, `{{ address.city }}`, `{{ address.country }}` )">
```

Coming from line:
 ```@click="radioAddressOnClick(`{{ address.name|escapejs }}`, `{{ address.delivery_address|escapejs }}`, `{{ address.postal_code }}`, `{{ address.city }}`, `{{ address.country }}` )">```

I have used backticks, `escapejs` filter and I've also made a custom filter:
```python
@register.filter
def escape_single_quotes(value):
    return value.replace("'", "\\'")    
```

When inspecting the elements tab in Chrome I get:
- with only back-ticks: ```radioAddressOnClick(`L'Address`, `2 rue de l'outil`, `12345`, `Brasov`, `Romania` )```
- with `escapejs` filter: ```radioAddressOnClick(`L\u0027Address`, `2 rue de l\u0027outil`, `12345`, `Brasov`, `Romania` )```
- with custom filter: ```radioAddressOnClick(`L\'Address`, `2 rue de l\'outil`, `12345`, `Brasov`, `Romania` )```

Yet I always get the same error:
![image](https://github.com/alpinejs/alpine/assets/39895513/3b3e74dd-bd8b-47bf-9559-4bdfeb7c3ca4)

**Comment by ekwoka** [comment]
the billing is on element with `billing-44` but you showed the code on `delivery-44`

**Reply by GeorgeFlorian** [reply]
Yes. I've noticed as soon as I've posted the issue. That is why I've closed it.
